examples/04_sweep_wind_directions.py

```diff
@@ -22,18 +22,15 @@
 """
 04_sweep_wind_directions
 
-This example demonstrates vectorization of wind direction.
-A vector of wind directions is passed to the intialize function
-and the powers of the two simulated turbines is computed for all
-wind directions in one call
+This example sweeps across wind directions while holding wind speed
+constant
 
 The power of both turbines for each wind direction is then plotted
 
 """
 
 # Instantiate FLORIS using either the GCH or CC model
```

Comment:
Update comment to reflect the line deleted below

Reply:
Tried to be more clear here